code/game/turfs/turf.dm

```diff
@@ -131,6 +131,20 @@
 	if (!mover)
 		return 1
 
+	if(mover.current_tethers.len)
```

Comment:
I'm not sure `Enter` is the best place to put this. Is there a reason not to be able to do it in `/atom/movable/Move()`?

Reply:
Does `Move()` also govern an atom being moved by an outside force, such as pulling?
A slave needs to not only be incapable of moving away from its master, but also *being moved* away from its master.

Reply:
`forceMove()` triggers `on_moved` hooks, so I'll put the logic into the tether datum.